app/code/core/Maho/Captcha/Helper/Data.php

```diff
@@ -100,7 +100,7 @@ public function verify(string $payload): bool
         try {
             $altcha = new Altcha\Altcha($this->getHmacKey());
             $isValid = $altcha->verifySolution($payload, true);
-            Mage::app()->getCache()->clean(Zend_Cache::CLEANING_MODE_OLD, [self::CACHE_TAG]);
+            Mage::app()->getCache()->clean([self::CACHE_TAG]);
```

Comment:
I think we want to actually remove this line, because otherwise it's clearing all captcha cache entries on each altcha attempt. The `self::CHALLENGE_EXPIRATION` on the next line will invalidate entries anyway.

Reply:
true, done it in https://github.com/MahoCommerce/maho/pull/165/commits/55ff79027c73aca809c907e30e766a6669917521, but I'm wondering how it was working before, only because we tested our single use and not concurrent use?

Reply:
I think it worked before because it used `CLEANING_MODE_OLD`, which would only prune expired entries. But I'm not sure that was ever necessary in the first place, maybe it was legacy code from before entries auto-expired?

Reply:
now I'm thinking all the CLEANING_MODE_OLD I've changed in this PR is actually wrong, I've to check them all

Reply:
actually there aren't any other CLEANING_MODE_OLD that were removed, better 😅